docs_user/modules/proc_adopting-the-networking-service.adoc

```diff
@@ -2,32 +2,31 @@
 
 = Adopting the {networking_service}
 
-Adopting {networking_first_ref} means that an existing `OpenStackControlPlane` custom resource (CR), where the {networking_service}
-is supposed to be disabled, should be patched to start the service with the
-configuration parameters provided by the source environment.
+To adopt the {networking_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR), where the {networking_service}
```

Comment:
maybe: "... you patch an existing `OpenStackControlPlane` custom resource (CR) that has the {networking_service} disabled." but also what is here already is ok.